tests/Bridge/NelmioApiDoc/Extractor/AnnotationsProvider/ApiPlatformProviderTest.php

```diff
@@ -32,6 +32,8 @@
 
 /**
  * @author Teoh Han Hui <teohhanhui@gmail.com>
+ *
+ * @group legacy
```

Comment:
We should also add a `@expectedDeprecation`.

Reply:
done